README.md.tmpl

```diff
@@ -0,0 +1,20 @@
+# {{ .AppName }}
+
+This API provides simple restful API access a service.
+
+## Endpoints
+
+```
+GET {{ .UrlBase }}/ping
+GET {{ .UrlBase }}/version
+GET {{ .UrlBase }}/metrics
+```
+
+## Authentication
+
+Authentication is accomplished via a pre-shared key.  This is done via the `X-Auth-Token` header.
```

Comment:
👀 Authentication is accomplished via an encrypted pre-shared key in the `X-Auth-Token` header